ci-operator/config/openshift-knative/net-istio/openshift-knative-net-istio-release-1.2.yaml

```diff
@@ -0,0 +1,53 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+  openshift_release_golang-1.17:
+    name: release
+    namespace: openshift
+    tag: golang-1.17
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.16
```

Comment:
This has to be 1.16 or 1.17? In previous versions we have this as 1.16.

Reply:
IMO would be nice if we align, no ?

Reply:
yes